pennylane/ops/qubit/parametric_ops.py

```diff
@@ -3028,3 +3028,144 @@ def adjoint(self):
 
     def pow(self, z):
         return [IsingXY(self.data[0] * z, wires=self.wires)]
+
+
+class PSWAP(Operation):
+    r"""Phase SWAP gate
+
+    .. math:: \mathtt{PSWAP}(\phi) = \begin{bmatrix}
```

Comment:
Let's adjust the design to how `qml.SWAP` renders.
```suggestion
    .. math:: PSWAP(\phi) = \begin{bmatrix}
```